content/docs/api-reference/read.md

```diff
@@ -79,8 +79,9 @@ These are loaded to memory directly (without using any disc space).
 
 ## Example: Load data from a DVC repository
 
-Any <abbr>data artifact</abbr> hosted online can be loaded directly in your
-Python code with this API. For example, let's say that you want to load and
+Any file tracked tracked in a <abbr>DVC project</abbr> (and
```

Comment:
tracked tracked

Reply:
Yes because you track in Git the data tracked in DVC...

Reply:
Jk, fixed in dac21d59.